src/SFA.DAS.ApplyService.Data/GatewayRepository.cs

```diff
@@ -119,14 +119,15 @@ public async Task<List<RoatpGatewaySummaryItem>> GetNewGatewayApplications(strin
                             apply.ApplicationStatus AS ApplicationStatus,
                             apply.GatewayReviewStatus AS GatewayReviewStatus,
                             apply.AssessorReviewStatus AS AssessorReviewStatus,
-                            apply.FinancialReviewStatus AS FinancialReviewStatus,
+                            fr.Status AS FinancialReviewStatus,
                             org.Name AS OrganisationName,
                             apply.UKPRN AS Ukprn,
                             JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ReferenceNumber') AS ApplicationReferenceNumber,
                             JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ProviderRouteName') AS ApplicationRoute,
                             JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ApplicationSubmittedOn') AS SubmittedDate
 	                      FROM Apply apply
 	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
+                          LEFT OUTER JOIN FinancialReview fr on fr.ApplicationId = apply.ApplicationId
```

Comment:
I wonder if we can save doing the left outer join here as we know PMO is after Gateway